apps/showcase/src/assets/trainings/sdk/steps/sdk-generation/instructions.html

```diff
@@ -0,0 +1,93 @@
+<p>
```

Comment:
We can have the generated sdk as a solution for this exercise? I am not sure this is useful, what do the others think?

Reply:
maybe a link to the `showcase-sdk` that is already versioned would be enough

Reply:
We could display the showcase sdk but there is a risk of the specification file not being aligned with the one in the Swagger Petstore